sbctl.go

```diff
@@ -41,9 +41,9 @@ func GetESP() (string, error) {
 	}
 
 	for _, location := range espLocations {
-		// "Touch" a file inside all candiadate locations to trigger an
+		// "Read" a file inside all candiadate locations to trigger an
 		// automount if there's an automount partition.
-		os.Stat(fmt.Sprintf("%s/does-not-exist", location))
+		_, _ = os.Stat(fmt.Sprintf("%s/does-not-exist", location))
```

Comment:
Whats the reason to define `_, _ =` in this case? Is this a linter complaint or a personal choice?

Reply:
Linter complaint: unchecked `err`.

It's almost always a very valid lint, this is a very unusual exception.

Reply:
Ahh! Makes sense :) Then it's fine